### PR TITLE
Feat: Button

### PR DESCRIPTION
**Pull request by vivekmehto**
https://user-images.githubusercontent.com/76930847/152111510-a72f7b66-3afc-4a74-8030-6d29842ab9b5.mov

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.btn {
+  margin: 0.5rem;
```
Comment (ttahm3d) [inline components/button/button.css]: Dont give margin and padding both inside component. Try to handle margin between elements using flex gap to the outer container

File: components/button/button.css [hunk 2]
```diff
@@ -55,7 +55,7 @@
 .btn-float {
   position: absolute;
```
Comment (ttahm3d) [inline components/button/button.css]: as far as I think this should be position fixed. absolute will move out of viewport on scroll

**Review by ttahm3d — COMMENTED** [review]
float btns should be fixed to viewport. Other than that not many changes

**Comment by ImrozNezami** [comment]
You can try to do changes mentioned above apart from that all looks good.

File: components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  padding: 0.3rem 0.5rem;
+  font-size: 0.9rem;
+}
+
+.btn-block {
+  width: 100%;
+}
+
+.btn-float {
+  position: absolute;
```
Comment (ImrozNezami) [inline components/button/button.css]: You can try to changes mentioned here apart from that all looks good.

**Review by ttahm3d — APPROVED** [review]

**Pull request merged**